<launch>
  <node name="ObjectDetection_node" pkg="ball_detect" type="ObjectDetection_node.py" output="screen">
    <!-- <param name="camera_id" value="/dev/camera" /> -->
    <param name="~camera_id" value="/dev/video0" />
    <param name="~yolo_model" value="$(find ball_detect)/scripts/best0703.pt" />
    <param name="~conf_threshold" value="0.6" />
  </node>
</launch>
